packages/base/src/3dview/mainview.tsx

```diff
@@ -668,28 +668,6 @@ export class MainView extends React.Component<IProps, IStates> {
     // Set the expoded view if it's enabled
     this._setupExplodedView();
 
-    // Clip plane rendering
-    const planeGeom = new THREE.PlaneGeometry(
-      this._refLength! * 10, // *10 is a bit arbitrary and extreme but that does not impact performance or anything
-      this._refLength! * 10
-    );
-    const planeMat = new THREE.MeshPhongMaterial({
-      color: DEFAULT_EDGE_COLOR,
-      stencilWrite: true,
-      stencilRef: 0,
-      stencilFunc: THREE.NotEqualStencilFunc,
-      stencilFail: THREE.ReplaceStencilOp,
-      stencilZFail: THREE.ReplaceStencilOp,
-      stencilZPass: THREE.ReplaceStencilOp,
-      side: THREE.DoubleSide,
-      wireframe: this.state.wireframe
-    });
-    this._clippingPlaneMesh = new THREE.Mesh(planeGeom, planeMat);
-    this._clippingPlaneMesh.onAfterRender = renderer => {
-      renderer.clearStencil();
-    };
-
-    this._scene.add(this._clippingPlaneMesh);
```

Comment:
Hi @martinRenou, removing this plane removes the default plane that's being rendered in the scene. I don't see any issue arised after removing this (especially in clipping), Are we good to remove this?

Reply:
Hi! 

That plane is used for filling the shapes cut by the clip plane:

![Screenshot from 2024-09-24 10-46-59](https://github.com/user-attachments/assets/87df0e38-cfaf-4d35-8dfc-7d9446d17265)

If we don't show it, the object will show as being non-solid once cut. So I believe we can't get rid of this code. I wonder if we should do `this._clippingPlaneMesh.visible = false` when clipping is not enabled, true otherwise.

Reply:
Thanks @martinRenou, I tried implementing your suggestion and now the _clipPlaneMesh is not visible by default at least but it doesn't look enough as in our case when we've clipping enabled we still do not have the whole clipPlaneMesh visible but it just covers the intersection of the mesh with clipPlane. 
But it's fully visible now: 
<img width="335" alt="image" src="https://github.com/user-attachments/assets/6752bed2-6ec8-4ee7-b9f1-7e5e6d258b95">

I'm on it trying debugging it but I believe your suggestion would accelerate the process :)

Reply:
It took me some time to figure out but was only a one line fix.